### PR TITLE
Fix #257 Frame aliasing `@graph` with omitGraph=true

### LINKED ISSUE #257: Frame aliasing `@graph` broken when `omitGraph = false`
**Describe the bug**
I want to rename the `@graph` name to something like `data` to have a Json output which can be consumed by applications that are not aware of Linked Data. This doesn't seem to work on titanium-json-ld

**To Reproduce**
Given this frame:
```json
{
  "@context": {
    "data": "@graph"
  }
}
```

and this Json-LD:
```json
{
  "@graph": [
    {
      "@id": "http://example.org/foo",
      "http://example.org/name": "Foo"
    },
    {
      "@id": "http://example.org/bar",
      "http://example.org/name": "Bar"
    }
  ]
}
```

Framing with the `omitGraph = false` option returns:

```json
{
  "@graph": [
    {
      "data": [
        {
          "@id": "http://example.org/foo",
          "http://example.org/name": "Foo"
        },
        {
          "@id": "http://example.org/bar",
          "http://example.org/name": "Bar"
        }
      ]
    }
  ],
  "@context": {
    "data": "@graph"
  }
}
```

Where the JSON-LD Playground returns:
```json
{
  "@context": {
    "data": "@graph"
  },
  "data": [
    {
      "@id": "http://example.org/bar",
      "http://example.org/name": "Bar"
    },
    {
      "@id": "http://example.org/foo",
      "http://example.org/name": "Foo"
    }
  ]
}
```

**Expected behavior**
My expectation is to have the same result as JSON-LD Playground

```json
{
  "@context": {
    "data": "@graph"
  },
  "data": [
    {
      "@id": "http://example.org/bar",
      "http://example.org/name": "Bar"
    },
    {
      "@id": "http://example.org/foo",
      "http://example.org/name": "Foo"
    }
  ]
}
```

### PR DESCRIPTION
**Pull request by lolgab**
Fixes #257

### CODE REVIEW

File: src/test/resources/com/apicatalog/jsonld/test/issue257-2-out.json
Comment (filip26) [inline src/test/resources/com/apicatalog/jsonld/test/issue257-2-out.json:1]: I'm not sure about this result, [please check that](https://json-ld.org/playground/#startTab=tab-framed&json-ld=%7B%22%40id%22%3A%22http%3A%2F%2Fexample.org%2Ffoo%22%2C%22http%3A%2F%2Fexample.org%2Fname%22%3A%22Foo%22%7D&frame=%7B%22%40context%22%3A%7B%22data%22%3A%22%40graph%22%7D%7D)

Comment (lolgab) [reply src/test/resources/com/apicatalog/jsonld/test/issue257-2-out.json:1]: If you disable `Compact Arrays` in JSON-LD Playground you also get a "data" array.
I assumed that `Compact Arrays` is the equivalent to our `omitGraph`.
I can add a test case with `omitGraph = true` if you want

Comment (filip26) [reply src/test/resources/com/apicatalog/jsonld/test/issue257-2-out.json:1]: I see, good job!

Comment (lolgab) [reply src/test/resources/com/apicatalog/jsonld/test/issue257-2-out.json:1]: Thank you! ❤️

**Comment by filip26** [comment]
It's about time to release a new version. The fix will be available in maven-central soon.

**Pull request merged**